README.md

```diff
@@ -328,6 +328,7 @@ conditions:
 | [`getResourceCondition`](example/functions/getResourceCondition) | Helper function to retreive conditions of resources          |
 | [`getComposedResource`](example/functions/getComposedResource)    | Helper function to retrieve observed composed resources      |
 | [`getCompositeResource`](example/functions/getCompositeResource) | Helper function to retreive the observed composite resource |
+| [`getExtraResources`](example/functions/getExtraResources)       | Helper function to retreive extra resources                  |
```

Comment:
```suggestion
| [`getExtraResources`](example/functions/getExtraResources)       | Helper function to retrieve extra resources                  |
```
nit - spelling - and I see that the others are mostly wrong too so if you want to fix those it would be great